projects/Apps/package.json

```diff
@@ -5,7 +5,8 @@
     "main": "index.js",
     "scripts": {
         "test": "yarn workspace @guardian/editions-common run test",
-        "build": "echo 'Apps does not need building.'"
+        "build": "echo 'Apps does not need building.'",
+        "postinstall": "npx jetifier"
```

Comment:
What is this doing?

Reply:
Converts old Android to AndroidX so things run.